roles/links-on-desktop/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Playbook to configure Postfix on a server to be able to send emails to admins
+
+- name: create links folder on UCS-Master
+  file:
+    path: /adfc/links/
+    state: directory
+    owner: root
+    group: root
+    mode: 0755
+  delegate_to: ucs-master.gst.hamburg.adfc.de
```

Comment:
Der Teil sollte in den Setup Master, oder? Dann muss er nicht jedes mal ausgeführt werden, wenn wir einen Client einrichten.

Reply:
Stimmt, das gehört eigentlich in setup-master.
Ich versuche noch eine Abfrage hinzuzufügen, ob der Ordner bereits vorhanden ist. Dann hätte man den Vorteil, dass die gesamte Funktionalität in der Rolle bleiben könnte.